src/test/java/myCollections/MyArrayListTest.java

```diff
@@ -0,0 +1,59 @@
+package myCollections;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class MyArrayListTest {
+
+    @Test
+    public void add() {
+        MyArrayList<String> list = new MyArrayList<String>();
```

Comment:
if you use java7+  MyArrayList<String> list = new MyArrayList<String>(); ->
MyArrayList<String> list = new MyArrayList<>(); 
Please check all your code.